### ISSUE TITLE
Calculate F1

### ISSUE THREAD

**Issue by Taluunas** [crux82/ganbert-pytorch#9]
Is it possible to calculate the F1-score of the model?
As far as I can tell currently only accuracy and loss is reported for the completed model.

**Comment by crux82** [comment]
Unfortunately not in the provided code. 

But you can just use the functions from scikit-learn

[https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html)

Hope it helps

Danilo

**Comment by shebna12** [comment]
> Is it possible to calculate the F1-score of the model? As far as I can tell currently only accuracy and loss is reported for the completed model.

@Taluunas Here's what I did to obtain some of the basic evaluation metrics of the trained model.
```
from sklearn.metrics import classification_report

print(classification_report(all_labels_ids, all_preds))
```

**Comment by Taluunas** [comment]
Thank you both, this was the approach I used. Thanks for confirmation.
